content/Wrath.SoundModTemplate/.template.config/template.json

```diff
@@ -45,5 +45,667 @@
 			"replaces": "{Game}",
 			"defaultValue": "Wrath"
 		}
-	}
+	},
+	"guids": [

```

Comment:
How does this work? See all guids, replace guids with new guids at template creation?

Reply:
Every instance of the listed GUID is replaced with a new randomly generated GUID that is consistent across all instances of the original.